bats_test/rename/rename-folder/rename_colliding_name.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Depends on ./rename_folder.sh
+
+rename_colliding_name() {
```

Comment:
Would it make more sense to let this file take an input argument for the old file name and then just use it as the input parameter to ardrive? Also, it looks like you're missing the -n flag.

Reply:
If so, then I would remove the file and directly use the `rename_folder.sh` from the `.bats` file which does exactly that

BTW What this method is running here is the case in which you try to rename the folder the same way as some sibling is (a file in this case). For renaming the folder into its current name there is `rename_same_name.sh`